components/ContentRenderer/ContentRenderer.tsx

```diff
@@ -30,13 +30,13 @@ import {
 
 import { Attachment, Gallery, Image, StoryBookmark, Variable } from './components';
 
-import styles from './SlateRenderer.module.scss';
+import styles from './ContentRenderer.module.scss';
 
 interface Props {
     nodes: Node | Node[];
 }
 
-function SlateRenderer({ nodes }: Props) {
+function ContentRenderer({ nodes }: Props) {
```

Comment:
I was hesitant to rename the file due to the amount of changes it would introduce in the PR but I think it makes it clearer now that we only have one way of rendering content.